packages/ui/react-ui-theme/src/styles/fragments/surface.ts

```diff
@@ -11,8 +11,13 @@
 // NOTE: This should align with theme's root --surface-bg.
 export const baseSurface = 'bg-neutral-12 dark:bg-neutral-900';
 
+export const baseSurfaceBorder = 'border-neutral-12 dark:border-neutral-900';
+
 // Sidebars, main heading (“topbar”), and nothing else.
-export const fixedSurface = 'bg-neutral-50/95 dark:bg-neutral-900/95 backdrop-blur';
+export const fixedSurface = 'bg-neutral-50/95 dark:bg-neutral-900/95 backdrop-blur-md';
+
+export const glassSurface =
```

Comment:
Please comment on what the fragment is for so it is easier to use and modify appropriately.